packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/calibrate-utils.ts

```diff
@@ -76,3 +79,49 @@ export function getErrorData(
 	});
 	return errors;
 }
+
+export const mapModelVarToDatasetVar = (state: CalibrationOperationStateCiemss, modelVariable: string) => {
+	if (modelVariable === 'timepoint_id') return state.timestampColName ?? '';
+	return state.mapping.find((d) => d.modelVariable === modelVariable)?.datasetVariable || '';
+};
+
+export function usePreparedChartInputs(
+	props: {
```

Comment:
It looks like this does not need this entire
```
 props: {
		node: WorkflowNode<CalibrationOperationStateCiemss>;
	}
```, but instead just utilizes the `calibrationId` and the `mapping` 

It might be nice to reduce the scope of what we are passing to this

Reply:
I do not understand the issue, what is wrong to change to two props instead of one? they will be reactive as well.

Reply:
It's either passing a single props as an argument vs multiple ractive/computed properties. 
```
usePreparedChartInputs(mapping: Ref<TypeofMapping>, state: Ref<typeofState>) { ... }

# In parent component
usePreparedChartInputs(computed(() => props.node.mapping), computed(() => props.node.state))
```
vs
```
usePreparedChartInputs(props: TypeofProps ) { ... }

# In parent component
usePreparedChartInputs(props)
```